pyproject.toml

```diff
@@ -59,12 +60,13 @@ doc = [
     "sphinx-design",
     "sphinx-examples",
     "sphinx-copybutton",
-    "sphinx-tabs",
+    "sphinx-tabs<=3.4.0",
```

Comment:
```suggestion
    "sphinx-tabs<=3.4.0", # sphinx-tabs 3.4.1 needs docutils >.17, which would conflict with our pin above
```

Reply:
all of this is because docutils has shipped a major bug in the last two releases that breaks bibliographies, we can delete when 0.2 comes out